README.md

```diff
@@ -1,9 +1,3 @@
-# Content Cache Operator
-
-A Juju charm for deploying and managing a content cache.
-
-## Overview
-
```

Comment:
I'd start with `A Juju charm for...` to match the other readmes. See https://github.com/canonical/indico-operator#readme
There's a comma missing: s/built on top of [Nginx](https://www.nginx.com/)
configurable/built on top of [Nginx](https://www.nginx.com/),
configurable